pwa-devdocs/src/_data/top-nav.yml

```diff
@@ -7,6 +7,9 @@
 - label: Peregrine Components
   url: /peregrine/
 
+- label: PageBuilder
```

Comment:
This should not be published yet, so this entry needs to be removed.

Reply:
@bdenham can you resolve this please.

Reply:
Yep. Got it. I will remove top level nav and move all page builder docs into _drafts immediately after the PR for PB-36 branch is finished (to make reviewing preview easier on reviewers).